### PR TITLE
no need for lambda in the case <fun>, ...

### PR DESCRIPTION
**Pull request by romainfrancois**
``` r
library(dplyr, warn.conflicts = FALSE)
library(dbplyr, warn.conflicts = FALSE)

tibble(x = 1:10) %>% 
  summarise_at("x", mean, na.rm = TRUE)
#> # A tibble: 1 x 1
#>       x
#>   <dbl>
#> 1   5.5


memdb_frame(x = 1:10) %>% 
  summarise_at("x", mean, na.rm = TRUE)
#> # Source:   lazy query [?? x 1]
#> # Database: sqlite 3.22.0 [:memory:]
#>       x
#>   <dbl>
#> 1   5.5
```

<sup>Created on 2019-04-02 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

### CODE REVIEW

File: R/funs.R [hunk 1]
```diff
@@ -102,11 +102,8 @@ as_fun_list <- function(.funs, .env, ...) {
         abort("not expecting this")
       }
       if (length(args)) {
-        # swoosh the extra arguments so that we have a function that only
-        # takes one argument `.`, or `.x` as a synonym
-        .x <- new_function(
-          list( . =missing_arg(), .x = sym(".")),
-          call2(.x, sym("."), !!!args),
+        .x <- new_quosure(
+          expr((!!.x)(., !!!args)),
```
Comment (romainfrancois) [inline R/funs.R]: so creating e.g. the quosure `mean(., na.rm = TRUE)` that is later `expr_substitute()`. This way `dbplyr` finds `mean`

Comment (lionel-) [reply R/funs.R]: Perhaps `call2(.x, quote(.), !!!args)` is more readable?

Does this mean we still have to quote the function in the colwise verbs?

Comment (romainfrancois) [reply R/funs.R]: I'm not entirely sure I understand, but I don't think so

**Comment by romainfrancois** [comment]
This fixes `cytominer` in the revdepcheck.

File: R/funs.R (hunk 1 above)
Comment (romainfrancois) [reply R/funs.R]: `.x` here is the function, not a symbol or a string. `dbplyr` picks that up in `fun_name()`

Comment (lionel-) [reply R/funs.R]: Will it still compute the mean in-database?

Comment (romainfrancois) [reply R/funs.R]: yes it will

**Pull request merged**